app/controllers/entry.rb

```diff
@@ -0,0 +1,73 @@
+include FileUtils::Verbose
+require 'fileutils'
+
+
+#create
+get '/upload' do
+  @cohorts    = Cohort.all
+  @languages  = Language.all
+  @phases     = Phase.all
```

Comment:
When your app is small this is fine, but eventually you might have a lot of data so loading all of your models into memory could be expensive. You could potentially use `.pluck` (http://apidock.com/rails/ActiveRecord/Calculations/pluck) to get only the data you need.